osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -116,7 +116,7 @@ protected override void LoadComplete()
             HitObject.DefaultsApplied += onDefaultsApplied;
 
             startTimeBindable = HitObject.StartTimeBindable.GetBoundCopy();
-            startTimeBindable.BindValueChanged(_ => updateState(ArmedState.Idle, true));
+            startTimeBindable.BindValueChanged(_ => updateState(State.Value, true));
```

Comment:
Threw me in a loop since I thought this was a no-op for a sec, but I missed the `force: true` so all good.